recipe/meta.yaml

```
@@ -1,34 +1,41 @@
 {% set name = "coin-or-utils" %}
-{% set version = "2.11.9" %}
+{% set version = "2.11.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/coin-or/CoinUtils/archive/releases/{{ version }}.tar.gz
-  sha256: 15d572ace4cd3b7c8ce117081b65a2bd5b5a4ebaba54fadc99c7a244160f88b8
+  sha256: 27da344479f38c82112d738501643dcb229e4ee96a5f87d4f406456bdc1b2cb4
   patches:
     - 0001-Patch-for-downstream.patch
 
 build:
   number: 0
-  skip: true  # [win]
   run_exports:
     - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - gnuconfig
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
-    - pkg-config
-    - make  # [unix]
+    - {{ compiler('fortran') }}  # [not win]
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - m2-make  # [win]
+    - m2-pkg-config  # [win]
+    - m2-grep  # [win]
+    - m2-sed  # [win]
+    - m2-bash  # [win]
+    - m2-base  # [win]
+    
   host:
     - zlib
     - bzip2
-    - libblas
+    - blas  # for includes and .lib?
```

you should use `blas-devel`, which contains the headers; `blas` is just a compatibility-wrapper around `blas-devel` for an old output.
```suggestion
    - blas-devel
```
```